presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -231,6 +232,8 @@
     public static final String PREFER_MERGE_JOIN = "prefer_merge_join";
     public static final String SEGMENTED_AGGREGATION_ENABLED = "segmented_aggregation_enabled";
     public static final String USE_EXTERNAL_PLAN_STATISTICS = "use_external_plan_statistics";
+    public static final String NODE_NUM_IN_LOGICAL_PLAN_EXCEED_THD_EXCEPTION_ENABLED = "node_num_in_logical_plan_exceed_thd_exception_enabled";
```

Comment:
We don't need this session property, if we just set max_node_num_in_logical_plan to 1e9, for example. We can just have this at default

Reply:
What is THD?

Reply:
What is thd?

Reply:
> We don't need this session property, if we just set max_node_num_in_logical_plan to 10^9, for example. We can just have this at default

I am afraid that 10^9 is too conservative, and we can already hit limit before that. Having it to be configurable gives more flexibility.

> What is thd?

I think we can add the number to logging (which is the next step after this PR), and get an idea on how large it is to set the threshold.

Reply:
I meant setting this limit to something large as 10^9 is same as setting enabled flag to false. However, I am fine with keeping it if its easier to manage.